README.md

```diff
@@ -1,7 +1,5 @@
 # AMQP 0-9-1 library and client for Node.JS
 
-[![Build Status](https://travis-ci.org/squaremo/amqp.node.png)](https://travis-ci.org/squaremo/amqp.node)
```

Comment:
You can use GA badge instead: 
`[![Build Status](https://github.com/amqp-node/amqplib/actions/workflows/test.yml/badge.svg)](https://github.com/amqp-node/amqplib/actions/workflows/test.yml)`